docs/modify/zodan/zodan_tutorial.md

```
@@ -1186,21 +1186,21 @@ $$;
 
 ### Show All Nodes
 
-Next, we will verify that all of the nodes are registered; the following 
+Next, we will verify that all of the nodes are registered; the following
 query displays all registered nodes:
 
 ```sql
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)